src/util/types.ts

```diff
@@ -78,6 +78,16 @@ export interface IEndpointTypes {
       students: IStudentInfo[];
     };
   };
+  "stats/projects": {
+    request: null;
+    response: {
```

Comment:
The response type is defined [here](https://github.com/kossiitkgp/KWoC-Backend/blob/8d311016f14795cf5dd1910b67321ec162b853c2/controllers/stats_project.go#L10-L18). It is an array of such objects.